src/index.ts

```diff
@@ -460,15 +460,26 @@ prog
     });
 
     const argv = process.argv.slice(2);
+    let jestConfig = {
+      ...createJestConfig(
+        relativePath => path.resolve(__dirname, '..', relativePath),
+        paths.appRoot
+      ),
+      ...appPackageJson.jest,
+    };
+    try {
+      // Allow overriding with jest.config
+      const jestConfigContents = require(path.resolve(
+        paths.appRoot,
+        'jest.config.js'
```

Comment:
We should add `jestConfig: resolveApp('./jest.config.js)` to `paths` and use it here.

Reply:
Yep, makes sense 👍